src/views/Home.jsx

```diff
@@ -10,15 +35,28 @@ export function Home({ data, setListPath }) {
 				{data &&
 					data.map((list) => {
 						return (
-							<SingleList
-								key={crypto.randomUUID()}
-								name={list.name}
-								setListPath={setListPath}
-								path={list.path}
-							/>
+							<>
+								<SingleList
+									key={crypto.randomUUID()}
+									name={list.name}
+									setListPath={setListPath}
+									path={list.path}
+								/>
+							</>
 						);
 					})}
 			</ul>
+			<form action="" onSubmit={handleSubmit}>
+				<label htmlFor="listName">Enter List Name:</label>
+				<input
+					type="text"
+					id="listName"
+					value={listName}
+					onChange={(e) => setListName(e.target.value)}
+				/>
+				<button>Submit</button>
+				<p>{error}</p>
```

Comment:
Great job! I like how the error is handled inside the async and await. Also noticed that when the form is successfully submitted and redirected to the List, the success message remains on the previous page, so the user doesn't get to see it.